changelog-tool/changelog.pony

```diff
@@ -79,25 +84,31 @@ class Release
     added = Section._empty(Added)
     changed = Section._empty(Changed)
 
+  fun ref add_entry(section_name: String, entry: String) ? =>
+    let section =
+      match section_name
+      | "fixed" =>
+        if fixed is None then fixed = Section._empty(Fixed) end
+        fixed as Section
+      | "added" =>
+        if added is None then added = Section._empty(Added) end
+        added as Section
+      | "changed" =>
+        if changed is None then changed = Section._empty(Changed) end
+        changed as Section
+      else error
+      end
+    section.entries.push("- " + entry)
```

Comment:
A newline should be added to the end of the entry if it doesn't already have one.

Reply:
Should this be done by using "String.contains" and appending `"\n"` or is there other idiomatic way of doing it?